### DISCUSSION TITLE
SN-solver in milonga still in Feenox?

### DISCUSSION THREAD

**Discussion by tzhang0475** [seamplex/feenox#10] [General]
Is the SN-solver from milonga still work in Feenox?

**Comment by gtheler** [comment]
Unfortunately not. The FEM formulation should be easy. The FVM will take more time because FeenoX now assumes finite elements only.
It's on the TO-DO list, but I'm not sure when it will be ported.

**Reply by tzhang0475** [reply]
Thx!

**Reply by gtheler** [reply]
If you have the time to take a stab at it, please create a pull request and I'll happily merge it. Otherwise I'll do it but I don't know when.

**Reply by gtheler** [reply]
Hey @tzhang0475 . At least for 1D, now FeenoX has support for neutron transport using SN. See the example at https://seamplex.com/feenox/examples/neutron_transport.html#reeds-problem

Feel free to try it out and see if you find any glitch.
Regards

**Reply by gtheler** [reply]
@tzhang0475 the SN solver using FEM is up and running, even working reasonably well in parallel. 
There are some examples in https://www.seamplex.com/feenox/examples/neutron_sn.html but I'll add more in due time.
You can also check https://seamplex.com/thesis/html/060-resultados/README.html (use some translation engine because the text is in Spanish).

**Reply by tzhang0475** [reply]
Hi, good to know the SN model is back in feenox, I ll take a look and there maybe questions coming! Thx!
